docs/integration/infra.md

```diff
@@ -0,0 +1,276 @@
+title: Infrastructure Monitoring Integration
+description: Infrastructure performance monitoring - metrics, events, processes, packages, log collection and parsing for bare-metal servers, virtual machines and containers with support for Docker, containerd, k8s, swarm, nomad, ecs, eks, aks, gke.
+
+The Sematext Infra Integration uses [Sematext Agent](../agents/sematext-agent) to collect metrics, events, processes, packages for bare-metal servers, virtual machines and containers. Container monitoring is supported for Docker, containerd, k8s, Swarm, Nomad, ECS, EKS, AKS, GKE and more.
```

Comment:
metrics, events, processes, packages for bare-metal servers, virtual machines and containers => metrics, events, processes, and package info for bare-metal servers, virtual machines and containers

ECS, EKS => AWS ECS, AWS EKS